### PR TITLE
PLAT-39440 emodule artifacts are broken for 5.2

### PR DESCRIPTION
**Pull request by mmasi-attivio**
On Windows, `env['PATH']` returns null because it's apparently doing case-sensitive comparison (variable name is `Path`). 

However, using `System.getenv('PATH')` _will_ ignore case and return the path properly. This means no changes need to be made on the Linux side.

### Test procedure
Perform on both Windows and Linux systems:
1. `mvn clean install`
1. `mvn archetype:generate -DarchetypeGroupId=com.attivio.platform.archetypes -DarchetypeArtifactId=attivio-archetype-module -DarchetypeVersion=5.2.6.1-SNAPSHOT`
     * Ensure archetype is generated successfully
1. `mvn archetype:generate -DarchetypeGroupId=com.attivio.platform.archetypes -DarchetypeArtifactId=attivio-archetype-client -DarchetypeVersion=5.2.6.1-SNAPSHOT`
     * Ensure archetype is generated successfully

Tested on both Windows and Linux with no issues.

Once approved this change will need to be merged into `develop` and `4.4`.

### CODE REVIEW

File: attivio-archetype-module/src/main/resources/META-INF/archetype-post-generate.groovy [hunk 1]
```diff
@@ -7,33 +7,28 @@ def webappsDir = new File(moduleDir, "src/main/resources/webapps/${artifactId}")
...
     dir.eachFileMatch(~/.*.java/) { file ->
-	String code = file.getText('UTF-8').replaceAll('com.sample.module', request.getGroupId()+'.'+request.getArtifactId())
-	file.newWriter().withWriter { w ->
-	    w << code
-	}
+        String code = file.getText('UTF-8').replaceAll('com.sample', request.getGroupId()+'.'+request.getArtifactId())
+        file.newWriter().withWriter { w ->
+            w << code
+        }
     }
```
Comment (serranom) [inline attivio-archetype-module/src/main/resources/META-INF/archetype-post-generate.groovy]: why did you strip out the .module above?

File: attivio-archetype-module/src/main/resources/META-INF/archetype-post-generate.groovy [hunk 2]
```diff
@@ -7,33 +7,28 @@ def webappsDir = new File(moduleDir, "src/main/resources/webapps/${artifactId}")
...
 }
-if (includeWeb && attivioHome == null) {
-    println "Web inclusion requires Attivio installation and Attivio was not found on path"
-    attivioHome = System.console().readLine 'Attivio Installation Directory?: '
-}
-if (attivioHome != null) {
-    println "Attivio installed at: ${attivioHome}"
-}
```
Comment (serranom) [inline attivio-archetype-module/src/main/resources/META-INF/archetype-post-generate.groovy]: Why were these removed?

**Review by serranom — CHANGES_REQUESTED** [review]

File: attivio-archetype-module/src/main/resources/META-INF/archetype-post-generate.groovy (hunk 2 above)
Comment (mmasi-attivio) [reply attivio-archetype-module/src/main/resources/META-INF/archetype-post-generate.groovy]: I think I got too greedy with my copy-paste -- everything is put back now.

**Review by serranom — APPROVED** [review]
